docs/README.md

```diff
@@ -7,6 +7,6 @@ Our documentation uses extended Markdown, as implemented by [MkDocs](http://mkdo
 
 - install MkDocs: `pip install mkdocs`
 - `cd` to the `docs/` folder and run:
-    - `python autogen.py`
```

Comment:
Sometimes, users forget to add Keras home directory to their path.

I suggest : 
```python
pip install mkdocs
export PYTHONPATH=$PYTHONPATH:$PWD
cd docs/
...
```

Reply:
Done. I prefered the `pip install -e .` to be consistent with the `keras/CONTRIBUTING.md` file.